### ISSUE TITLE
Migrate applicant_application_statuses.test.ts to North Star

### ISSUE THREAD

**Issue by ssandbekkhaug** [civiform/civiform#8907]
# Instructions
See https://github.com/civiform/civiform/issues/8791

# Notes
This one looks pretty easy. There's only 1 test case, and the North Star version already exists.

The only thing "missing" is that the NS test case does not validate a11y.
